website/content/api-docs/acl/index.mdx

```diff
@@ -16,7 +16,7 @@ the [ACL tutorial](https://learn.hashicorp.com/tutorials/consul/access-control-s
 ## Bootstrap ACLs
 
 This endpoint does a special one-time bootstrap of the ACL system, making the first
-management token if the [`acl.tokens.master`](/docs/agent/options#acl_tokens_master)
+management token if the [`acl.tokens.initial_management`](/docs/agent/options#acl_tokens_initial_management)
```

Comment:
Is the intent that, if someone is on Consul 1.11, they will follow this link, see

> This is available in Consul 1.11 and later. In prior versions, use [`acl.tokens.master`](#acl_tokens_master)

and then find out the correct (previous) name that way?

I imagine we've had this problem before (and will have new instances of this problem in the future) when we rename things... do we mention every name everywhere? Or mention just the most recent name in most docs, but link to information about old names?

[Long term, this might be best solved by having versioned docs pages. But that's well outside the scope of this PR :) ]

Reply:
Yep! In this case, it's a bit more onerous because we've already renamed the token before (`acl_master_token` ⇒ `acl.tokens.master` ⇒ `acl.tokens.initial_management`) so users on very old versions have a bit of a trail to follow.

> do we mention every name everywhere? Or mention just the most recent name in most docs

That seems to be the case, yeah. For example, the current ACL System page [mentions](https://www.consul.io/docs/security/acl/acl-system#acl-agent-master-token) `acl.tokens.agent_master`, not `acl_agent_master_token`.